File: ironclient/lib/models/officialnotification.dart
class OfficialNotification {
  String? id;
  String title;
  String message;
  bool enabled;

  OfficialNotification({
    this.id,
    required this.title,
    required this.message,
    required this.enabled,
  });

  factory OfficialNotification.fromJson(Map<String, dynamic> jsonMap) =>
      OfficialNotification(
        id: jsonMap['_id'],
        title: jsonMap['title'],
        message: jsonMap['message'],
        enabled: jsonMap['enabled']
      );

  Map<String, dynamic> toJson() => {
    '_id': id,
    'title': title,
    'message': message,
    'enabled': enabled,
  };
}

File: ironapi/models/officialnotification.js
const mongoose = require('mongoose');

var OfficialNotificationSchema = new mongoose.Schema({  
    title: {type: String},
    message: {type: String},
    enabled: {type: Boolean, default: true},
}, { timestamps: { createdAt: 'created', updatedAt: 'lastUpdate'}}, { collection: 'officialnotification' });

mongoose.model('OfficialNotification', OfficialNotificationSchema);

module.exports = mongoose.model('OfficialNotification');